
[34m[1mwandb[39m[22m: [33mWARNING[39m The save_model argument by default saves the model in the HDF5 format that cannot save custom objects like subclassed models and custom layers. This behavior will be deprecated in a future release in favor of the SavedModel format. Meanwhile, the HDF5 model is saved as W&B files and the SavedModel as W&B Artifacts.
Epoch 1/100
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26493.8262 - RMSE_step_t_humidity: 281.7759 - RMSE_step_t_1_humidity: 281.7524 - RMSE_step_t_wind_dir: 3.8893 - RMSE_step_t_1_wind_dir: 4.0445 - RMSE_step_t_wind_speed: 2.7680 - RMSE_step_t_1_wind_speed: 2.9686
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 10s 562ms/step - loss: 26493.8262 - RMSE_step_t_humidity: 281.7759 - RMSE_step_t_1_humidity: 281.7524 - RMSE_step_t_wind_dir: 3.8893 - RMSE_step_t_1_wind_dir: 4.0445 - RMSE_step_t_wind_speed: 2.7680 - RMSE_step_t_1_wind_speed: 2.9686 - val_loss: 26539.4355 - val_RMSE_step_t_humidity: 282.5806 - val_RMSE_step_t_1_humidity: 282.4544 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5782
Epoch 2/100
10/10 [==============================] - ETA: 0s - loss: 26493.8262 - RMSE_step_t_humidity: 281.8648 - RMSE_step_t_1_humidity: 281.8442 - RMSE_step_t_wind_dir: 3.8770 - RMSE_step_t_1_wind_dir: 4.0598 - RMSE_step_t_wind_speed: 2.7833 - RMSE_step_t_1_wind_speed: 2.9926
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 476ms/step - loss: 26493.8262 - RMSE_step_t_humidity: 281.8648 - RMSE_step_t_1_humidity: 281.8442 - RMSE_step_t_wind_dir: 3.8770 - RMSE_step_t_1_wind_dir: 4.0598 - RMSE_step_t_wind_speed: 2.7833 - RMSE_step_t_1_wind_speed: 2.9926 - val_loss: 26539.4316 - val_RMSE_step_t_humidity: 282.5806 - val_RMSE_step_t_1_humidity: 282.4544 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5782
Epoch 3/100
10/10 [==============================] - ETA: 0s - loss: 26494.2793 - RMSE_step_t_humidity: 281.7509 - RMSE_step_t_1_humidity: 281.7379 - RMSE_step_t_wind_dir: 3.8869 - RMSE_step_t_1_wind_dir: 4.0613 - RMSE_step_t_wind_speed: 2.7752 - RMSE_step_t_1_wind_speed: 2.9986
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 410ms/step - loss: 26494.2793 - RMSE_step_t_humidity: 281.7509 - RMSE_step_t_1_humidity: 281.7379 - RMSE_step_t_wind_dir: 3.8869 - RMSE_step_t_1_wind_dir: 4.0613 - RMSE_step_t_wind_speed: 2.7752 - RMSE_step_t_1_wind_speed: 2.9986 - val_loss: 26539.4238 - val_RMSE_step_t_humidity: 282.5805 - val_RMSE_step_t_1_humidity: 282.4543 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5782
Epoch 4/100
10/10 [==============================] - ETA: 0s - loss: 26493.3730 - RMSE_step_t_humidity: 281.7487 - RMSE_step_t_1_humidity: 281.7504 - RMSE_step_t_wind_dir: 3.8866 - RMSE_step_t_1_wind_dir: 4.0581 - RMSE_step_t_wind_speed: 2.7691 - RMSE_step_t_1_wind_speed: 2.9636
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 445ms/step - loss: 26493.3730 - RMSE_step_t_humidity: 281.7487 - RMSE_step_t_1_humidity: 281.7504 - RMSE_step_t_wind_dir: 3.8866 - RMSE_step_t_1_wind_dir: 4.0581 - RMSE_step_t_wind_speed: 2.7691 - RMSE_step_t_1_wind_speed: 2.9636 - val_loss: 26539.4199 - val_RMSE_step_t_humidity: 282.5805 - val_RMSE_step_t_1_humidity: 282.4543 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 5/100
10/10 [==============================] - ETA: 0s - loss: 26493.0879 - RMSE_step_t_humidity: 281.8339 - RMSE_step_t_1_humidity: 281.8338 - RMSE_step_t_wind_dir: 3.8258 - RMSE_step_t_1_wind_dir: 3.9746 - RMSE_step_t_wind_speed: 2.7419 - RMSE_step_t_1_wind_speed: 2.9438
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 439ms/step - loss: 26493.0879 - RMSE_step_t_humidity: 281.8339 - RMSE_step_t_1_humidity: 281.8338 - RMSE_step_t_wind_dir: 3.8258 - RMSE_step_t_1_wind_dir: 3.9746 - RMSE_step_t_wind_speed: 2.7419 - RMSE_step_t_1_wind_speed: 2.9438 - val_loss: 26539.4082 - val_RMSE_step_t_humidity: 282.5804 - val_RMSE_step_t_1_humidity: 282.4543 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 6/100
 9/10 [==========================>...] - ETA: 0s - loss: 26497.8672 - RMSE_step_t_humidity: 281.8754 - RMSE_step_t_1_humidity: 281.8483 - RMSE_step_t_wind_dir: 3.8793 - RMSE_step_t_1_wind_dir: 4.0670 - RMSE_step_t_wind_speed: 2.8012 - RMSE_step_t_1_wind_speed: 3.0275
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 491ms/step - loss: 26494.2500 - RMSE_step_t_humidity: 281.8063 - RMSE_step_t_1_humidity: 281.8004 - RMSE_step_t_wind_dir: 3.8169 - RMSE_step_t_1_wind_dir: 3.9986 - RMSE_step_t_wind_speed: 2.7606 - RMSE_step_t_1_wind_speed: 2.9901 - val_loss: 26539.4043 - val_RMSE_step_t_humidity: 282.5804 - val_RMSE_step_t_1_humidity: 282.4542 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 7/100
10/10 [==============================] - ETA: 0s - loss: 26493.8262 - RMSE_step_t_humidity: 281.7542 - RMSE_step_t_1_humidity: 281.7208 - RMSE_step_t_wind_dir: 3.8032 - RMSE_step_t_1_wind_dir: 4.0100 - RMSE_step_t_wind_speed: 2.7340 - RMSE_step_t_1_wind_speed: 2.9676
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 416ms/step - loss: 26493.8262 - RMSE_step_t_humidity: 281.7542 - RMSE_step_t_1_humidity: 281.7208 - RMSE_step_t_wind_dir: 3.8032 - RMSE_step_t_1_wind_dir: 4.0100 - RMSE_step_t_wind_speed: 2.7340 - RMSE_step_t_1_wind_speed: 2.9676 - val_loss: 26539.3984 - val_RMSE_step_t_humidity: 282.5804 - val_RMSE_step_t_1_humidity: 282.4542 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 8/100
10/10 [==============================] - ETA: 0s - loss: 26494.4766 - RMSE_step_t_humidity: 281.7438 - RMSE_step_t_1_humidity: 281.7075 - RMSE_step_t_wind_dir: 3.8071 - RMSE_step_t_1_wind_dir: 3.9651 - RMSE_step_t_wind_speed: 2.7436 - RMSE_step_t_1_wind_speed: 2.9602
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 495ms/step - loss: 26494.4766 - RMSE_step_t_humidity: 281.7438 - RMSE_step_t_1_humidity: 281.7075 - RMSE_step_t_wind_dir: 3.8071 - RMSE_step_t_1_wind_dir: 3.9651 - RMSE_step_t_wind_speed: 2.7436 - RMSE_step_t_1_wind_speed: 2.9602 - val_loss: 26539.3906 - val_RMSE_step_t_humidity: 282.5804 - val_RMSE_step_t_1_humidity: 282.4542 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 9/100
 9/10 [==========================>...] - ETA: 0s - loss: 26496.6797 - RMSE_step_t_humidity: 281.8716 - RMSE_step_t_1_humidity: 281.8403 - RMSE_step_t_wind_dir: 3.8503 - RMSE_step_t_1_wind_dir: 4.0236 - RMSE_step_t_wind_speed: 2.7853 - RMSE_step_t_1_wind_speed: 3.0158
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 406ms/step - loss: 26494.6445 - RMSE_step_t_humidity: 281.8185 - RMSE_step_t_1_humidity: 281.8257 - RMSE_step_t_wind_dir: 3.8697 - RMSE_step_t_1_wind_dir: 4.0548 - RMSE_step_t_wind_speed: 2.7879 - RMSE_step_t_1_wind_speed: 3.0079 - val_loss: 26539.3848 - val_RMSE_step_t_humidity: 282.5803 - val_RMSE_step_t_1_humidity: 282.4542 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 10/100
10/10 [==============================] - 4s 448ms/step - loss: 26494.4512 - RMSE_step_t_humidity: 281.9697 - RMSE_step_t_1_humidity: 281.9141 - RMSE_step_t_wind_dir: 3.9058 - RMSE_step_t_1_wind_dir: 4.0712 - RMSE_step_t_wind_speed: 2.8207 - RMSE_step_t_1_wind_speed: 3.0106 - val_loss: 26539.3770 - val_RMSE_step_t_humidity: 282.5803 - val_RMSE_step_t_1_humidity: 282.4541 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 11/100
10/10 [==============================] - 0s 28ms/step - loss: 26494.7734 - RMSE_step_t_humidity: 281.9131 - RMSE_step_t_1_humidity: 281.9055 - RMSE_step_t_wind_dir: 3.9115 - RMSE_step_t_1_wind_dir: 4.0757 - RMSE_step_t_wind_speed: 2.8024 - RMSE_step_t_1_wind_speed: 3.0321 - val_loss: 26539.3770 - val_RMSE_step_t_humidity: 282.5802 - val_RMSE_step_t_1_humidity: 282.4541 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 12/100
10/10 [==============================] - ETA: 0s - loss: 26493.7266 - RMSE_step_t_humidity: 281.7335 - RMSE_step_t_1_humidity: 281.7031 - RMSE_step_t_wind_dir: 3.7938 - RMSE_step_t_1_wind_dir: 3.9696 - RMSE_step_t_wind_speed: 2.7331 - RMSE_step_t_1_wind_speed: 2.9548
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 447ms/step - loss: 26493.7266 - RMSE_step_t_humidity: 281.7335 - RMSE_step_t_1_humidity: 281.7031 - RMSE_step_t_wind_dir: 3.7938 - RMSE_step_t_1_wind_dir: 3.9696 - RMSE_step_t_wind_speed: 2.7331 - RMSE_step_t_1_wind_speed: 2.9548 - val_loss: 26539.3691 - val_RMSE_step_t_humidity: 282.5802 - val_RMSE_step_t_1_humidity: 282.4540 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 13/100
10/10 [==============================] - ETA: 0s - loss: 26493.5762 - RMSE_step_t_humidity: 281.8150 - RMSE_step_t_1_humidity: 281.7778 - RMSE_step_t_wind_dir: 3.8187 - RMSE_step_t_1_wind_dir: 3.9966 - RMSE_step_t_wind_speed: 2.7912 - RMSE_step_t_1_wind_speed: 2.9990
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 409ms/step - loss: 26493.5762 - RMSE_step_t_humidity: 281.8150 - RMSE_step_t_1_humidity: 281.7778 - RMSE_step_t_wind_dir: 3.8187 - RMSE_step_t_1_wind_dir: 3.9966 - RMSE_step_t_wind_speed: 2.7912 - RMSE_step_t_1_wind_speed: 2.9990 - val_loss: 26539.3594 - val_RMSE_step_t_humidity: 282.5802 - val_RMSE_step_t_1_humidity: 282.4540 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 14/100
10/10 [==============================] - ETA: 0s - loss: 26493.2617 - RMSE_step_t_humidity: 281.8746 - RMSE_step_t_1_humidity: 281.8730 - RMSE_step_t_wind_dir: 3.9038 - RMSE_step_t_1_wind_dir: 4.0668 - RMSE_step_t_wind_speed: 2.7796 - RMSE_step_t_1_wind_speed: 2.9920
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 5s 507ms/step - loss: 26493.2617 - RMSE_step_t_humidity: 281.8746 - RMSE_step_t_1_humidity: 281.8730 - RMSE_step_t_wind_dir: 3.9038 - RMSE_step_t_1_wind_dir: 4.0668 - RMSE_step_t_wind_speed: 2.7796 - RMSE_step_t_1_wind_speed: 2.9920 - val_loss: 26539.3535 - val_RMSE_step_t_humidity: 282.5802 - val_RMSE_step_t_1_humidity: 282.4540 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 15/100
 8/10 [=======================>......] - ETA: 0s - loss: 26495.7148 - RMSE_step_t_humidity: 281.8617 - RMSE_step_t_1_humidity: 281.8362 - RMSE_step_t_wind_dir: 3.9047 - RMSE_step_t_1_wind_dir: 4.0557 - RMSE_step_t_wind_speed: 2.8368 - RMSE_step_t_1_wind_speed: 3.0303
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 410ms/step - loss: 26492.9922 - RMSE_step_t_humidity: 281.8885 - RMSE_step_t_1_humidity: 281.8219 - RMSE_step_t_wind_dir: 3.8383 - RMSE_step_t_1_wind_dir: 4.0402 - RMSE_step_t_wind_speed: 2.7718 - RMSE_step_t_1_wind_speed: 3.0071 - val_loss: 26539.3457 - val_RMSE_step_t_humidity: 282.5801 - val_RMSE_step_t_1_humidity: 282.4539 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6042 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 16/100
 8/10 [=======================>......] - ETA: 0s - loss: 26509.8867 - RMSE_step_t_humidity: 281.9264 - RMSE_step_t_1_humidity: 281.9221 - RMSE_step_t_wind_dir: 3.9109 - RMSE_step_t_1_wind_dir: 4.0956 - RMSE_step_t_wind_speed: 2.8227 - RMSE_step_t_1_wind_speed: 3.0678
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 454ms/step - loss: 26494.2168 - RMSE_step_t_humidity: 281.7744 - RMSE_step_t_1_humidity: 281.7649 - RMSE_step_t_wind_dir: 3.8549 - RMSE_step_t_1_wind_dir: 4.0572 - RMSE_step_t_wind_speed: 2.7561 - RMSE_step_t_1_wind_speed: 3.0022 - val_loss: 26539.3418 - val_RMSE_step_t_humidity: 282.5801 - val_RMSE_step_t_1_humidity: 282.4539 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 17/100
 9/10 [==========================>...] - ETA: 0s - loss: 26491.8613 - RMSE_step_t_humidity: 281.8327 - RMSE_step_t_1_humidity: 281.8271 - RMSE_step_t_wind_dir: 3.8285 - RMSE_step_t_1_wind_dir: 4.0223 - RMSE_step_t_wind_speed: 2.7587 - RMSE_step_t_1_wind_speed: 3.0067
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 7s 737ms/step - loss: 26493.8281 - RMSE_step_t_humidity: 281.8912 - RMSE_step_t_1_humidity: 281.8315 - RMSE_step_t_wind_dir: 3.8790 - RMSE_step_t_1_wind_dir: 4.0690 - RMSE_step_t_wind_speed: 2.8117 - RMSE_step_t_1_wind_speed: 3.0309 - val_loss: 26539.3379 - val_RMSE_step_t_humidity: 282.5800 - val_RMSE_step_t_1_humidity: 282.4539 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 18/100
10/10 [==============================] - ETA: 0s - loss: 26493.9492 - RMSE_step_t_humidity: 281.8244 - RMSE_step_t_1_humidity: 281.8546 - RMSE_step_t_wind_dir: 3.8918 - RMSE_step_t_1_wind_dir: 4.0996 - RMSE_step_t_wind_speed: 2.7954 - RMSE_step_t_1_wind_speed: 3.0384
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 456ms/step - loss: 26493.9492 - RMSE_step_t_humidity: 281.8244 - RMSE_step_t_1_humidity: 281.8546 - RMSE_step_t_wind_dir: 3.8918 - RMSE_step_t_1_wind_dir: 4.0996 - RMSE_step_t_wind_speed: 2.7954 - RMSE_step_t_1_wind_speed: 3.0384 - val_loss: 26539.3281 - val_RMSE_step_t_humidity: 282.5800 - val_RMSE_step_t_1_humidity: 282.4539 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 19/100
10/10 [==============================] - ETA: 0s - loss: 26493.4375 - RMSE_step_t_humidity: 281.9854 - RMSE_step_t_1_humidity: 281.9309 - RMSE_step_t_wind_dir: 3.9073 - RMSE_step_t_1_wind_dir: 4.0548 - RMSE_step_t_wind_speed: 2.8328 - RMSE_step_t_1_wind_speed: 3.0165
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 434ms/step - loss: 26493.4375 - RMSE_step_t_humidity: 281.9854 - RMSE_step_t_1_humidity: 281.9309 - RMSE_step_t_wind_dir: 3.9073 - RMSE_step_t_1_wind_dir: 4.0548 - RMSE_step_t_wind_speed: 2.8328 - RMSE_step_t_1_wind_speed: 3.0165 - val_loss: 26539.3223 - val_RMSE_step_t_humidity: 282.5800 - val_RMSE_step_t_1_humidity: 282.4538 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5781
Epoch 20/100
10/10 [==============================] - ETA: 0s - loss: 26493.6992 - RMSE_step_t_humidity: 281.9865 - RMSE_step_t_1_humidity: 281.9797 - RMSE_step_t_wind_dir: 3.9145 - RMSE_step_t_1_wind_dir: 4.0300 - RMSE_step_t_wind_speed: 2.8084 - RMSE_step_t_1_wind_speed: 2.9883
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 447ms/step - loss: 26493.6992 - RMSE_step_t_humidity: 281.9865 - RMSE_step_t_1_humidity: 281.9797 - RMSE_step_t_wind_dir: 3.9145 - RMSE_step_t_1_wind_dir: 4.0300 - RMSE_step_t_wind_speed: 2.8084 - RMSE_step_t_1_wind_speed: 2.9883 - val_loss: 26539.3184 - val_RMSE_step_t_humidity: 282.5800 - val_RMSE_step_t_1_humidity: 282.4538 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 21/100
 9/10 [==========================>...] - ETA: 0s - loss: 26484.2754 - RMSE_step_t_humidity: 281.8009 - RMSE_step_t_1_humidity: 281.7791 - RMSE_step_t_wind_dir: 3.8155 - RMSE_step_t_1_wind_dir: 3.9965 - RMSE_step_t_wind_speed: 2.7559 - RMSE_step_t_1_wind_speed: 2.9918
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 453ms/step - loss: 26493.8633 - RMSE_step_t_humidity: 281.9574 - RMSE_step_t_1_humidity: 281.9335 - RMSE_step_t_wind_dir: 3.9206 - RMSE_step_t_1_wind_dir: 4.1096 - RMSE_step_t_wind_speed: 2.8263 - RMSE_step_t_1_wind_speed: 3.0543 - val_loss: 26539.3105 - val_RMSE_step_t_humidity: 282.5799 - val_RMSE_step_t_1_humidity: 282.4537 - val_RMSE_step_t_wind_dir: 2.9126 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 22/100
 9/10 [==========================>...] - ETA: 0s - loss: 26507.8008 - RMSE_step_t_humidity: 281.9275 - RMSE_step_t_1_humidity: 281.9012 - RMSE_step_t_wind_dir: 3.8619 - RMSE_step_t_1_wind_dir: 4.0405 - RMSE_step_t_wind_speed: 2.7913 - RMSE_step_t_1_wind_speed: 3.0243
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 417ms/step - loss: 26493.5918 - RMSE_step_t_humidity: 281.6850 - RMSE_step_t_1_humidity: 281.6755 - RMSE_step_t_wind_dir: 3.8160 - RMSE_step_t_1_wind_dir: 3.9940 - RMSE_step_t_wind_speed: 2.7527 - RMSE_step_t_1_wind_speed: 2.9652 - val_loss: 26539.3027 - val_RMSE_step_t_humidity: 282.5799 - val_RMSE_step_t_1_humidity: 282.4537 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 23/100
10/10 [==============================] - ETA: 0s - loss: 26494.0078 - RMSE_step_t_humidity: 281.7693 - RMSE_step_t_1_humidity: 281.7151 - RMSE_step_t_wind_dir: 3.8436 - RMSE_step_t_1_wind_dir: 4.0201 - RMSE_step_t_wind_speed: 2.7934 - RMSE_step_t_1_wind_speed: 3.0079
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 448ms/step - loss: 26494.0078 - RMSE_step_t_humidity: 281.7693 - RMSE_step_t_1_humidity: 281.7151 - RMSE_step_t_wind_dir: 3.8436 - RMSE_step_t_1_wind_dir: 4.0201 - RMSE_step_t_wind_speed: 2.7934 - RMSE_step_t_1_wind_speed: 3.0079 - val_loss: 26539.2969 - val_RMSE_step_t_humidity: 282.5799 - val_RMSE_step_t_1_humidity: 282.4537 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 24/100
10/10 [==============================] - ETA: 0s - loss: 26492.9785 - RMSE_step_t_humidity: 281.9535 - RMSE_step_t_1_humidity: 281.8892 - RMSE_step_t_wind_dir: 3.8555 - RMSE_step_t_1_wind_dir: 4.0283 - RMSE_step_t_wind_speed: 2.8089 - RMSE_step_t_1_wind_speed: 3.0370
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 452ms/step - loss: 26492.9785 - RMSE_step_t_humidity: 281.9535 - RMSE_step_t_1_humidity: 281.8892 - RMSE_step_t_wind_dir: 3.8555 - RMSE_step_t_1_wind_dir: 4.0283 - RMSE_step_t_wind_speed: 2.8089 - RMSE_step_t_1_wind_speed: 3.0370 - val_loss: 26539.2910 - val_RMSE_step_t_humidity: 282.5798 - val_RMSE_step_t_1_humidity: 282.4536 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 25/100
10/10 [==============================] - 4s 435ms/step - loss: 26493.3340 - RMSE_step_t_humidity: 281.6972 - RMSE_step_t_1_humidity: 281.7078 - RMSE_step_t_wind_dir: 3.7716 - RMSE_step_t_1_wind_dir: 3.9547 - RMSE_step_t_wind_speed: 2.7440 - RMSE_step_t_1_wind_speed: 2.9536 - val_loss: 26539.2832 - val_RMSE_step_t_humidity: 282.5798 - val_RMSE_step_t_1_humidity: 282.4536 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 26/100
10/10 [==============================] - ETA: 0s - loss: 26492.6621 - RMSE_step_t_humidity: 281.9777 - RMSE_step_t_1_humidity: 281.9023 - RMSE_step_t_wind_dir: 3.8972 - RMSE_step_t_1_wind_dir: 4.0702 - RMSE_step_t_wind_speed: 2.8080 - RMSE_step_t_1_wind_speed: 3.0299
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 429ms/step - loss: 26492.6621 - RMSE_step_t_humidity: 281.9777 - RMSE_step_t_1_humidity: 281.9023 - RMSE_step_t_wind_dir: 3.8972 - RMSE_step_t_1_wind_dir: 4.0702 - RMSE_step_t_wind_speed: 2.8080 - RMSE_step_t_1_wind_speed: 3.0299 - val_loss: 26539.2812 - val_RMSE_step_t_humidity: 282.5797 - val_RMSE_step_t_1_humidity: 282.4536 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 27/100
10/10 [==============================] - ETA: 0s - loss: 26493.2227 - RMSE_step_t_humidity: 281.9209 - RMSE_step_t_1_humidity: 281.8887 - RMSE_step_t_wind_dir: 3.8872 - RMSE_step_t_1_wind_dir: 4.0738 - RMSE_step_t_wind_speed: 2.7875 - RMSE_step_t_1_wind_speed: 3.0031
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 20s 2s/step - loss: 26493.2227 - RMSE_step_t_humidity: 281.9209 - RMSE_step_t_1_humidity: 281.8887 - RMSE_step_t_wind_dir: 3.8872 - RMSE_step_t_1_wind_dir: 4.0738 - RMSE_step_t_wind_speed: 2.7875 - RMSE_step_t_1_wind_speed: 3.0031 - val_loss: 26539.2754 - val_RMSE_step_t_humidity: 282.5797 - val_RMSE_step_t_1_humidity: 282.4536 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 28/100
10/10 [==============================] - ETA: 0s - loss: 26492.9277 - RMSE_step_t_humidity: 282.0069 - RMSE_step_t_1_humidity: 281.9848 - RMSE_step_t_wind_dir: 3.8716 - RMSE_step_t_1_wind_dir: 4.0263 - RMSE_step_t_wind_speed: 2.8170 - RMSE_step_t_1_wind_speed: 3.0440
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 406ms/step - loss: 26492.9277 - RMSE_step_t_humidity: 282.0069 - RMSE_step_t_1_humidity: 281.9848 - RMSE_step_t_wind_dir: 3.8716 - RMSE_step_t_1_wind_dir: 4.0263 - RMSE_step_t_wind_speed: 2.8170 - RMSE_step_t_1_wind_speed: 3.0440 - val_loss: 26539.2676 - val_RMSE_step_t_humidity: 282.5797 - val_RMSE_step_t_1_humidity: 282.4535 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 29/100
10/10 [==============================] - ETA: 0s - loss: 26493.6660 - RMSE_step_t_humidity: 281.7437 - RMSE_step_t_1_humidity: 281.7628 - RMSE_step_t_wind_dir: 3.8764 - RMSE_step_t_1_wind_dir: 4.0643 - RMSE_step_t_wind_speed: 2.8044 - RMSE_step_t_1_wind_speed: 3.0172
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 442ms/step - loss: 26493.6660 - RMSE_step_t_humidity: 281.7437 - RMSE_step_t_1_humidity: 281.7628 - RMSE_step_t_wind_dir: 3.8764 - RMSE_step_t_1_wind_dir: 4.0643 - RMSE_step_t_wind_speed: 2.8044 - RMSE_step_t_1_wind_speed: 3.0172 - val_loss: 26539.2598 - val_RMSE_step_t_humidity: 282.5797 - val_RMSE_step_t_1_humidity: 282.4535 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 30/100
10/10 [==============================] - ETA: 0s - loss: 26491.6738 - RMSE_step_t_humidity: 281.9301 - RMSE_step_t_1_humidity: 281.9281 - RMSE_step_t_wind_dir: 3.9052 - RMSE_step_t_1_wind_dir: 4.0801 - RMSE_step_t_wind_speed: 2.7743 - RMSE_step_t_1_wind_speed: 3.0073
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 429ms/step - loss: 26491.6738 - RMSE_step_t_humidity: 281.9301 - RMSE_step_t_1_humidity: 281.9281 - RMSE_step_t_wind_dir: 3.9052 - RMSE_step_t_1_wind_dir: 4.0801 - RMSE_step_t_wind_speed: 2.7743 - RMSE_step_t_1_wind_speed: 3.0073 - val_loss: 26539.2520 - val_RMSE_step_t_humidity: 282.5796 - val_RMSE_step_t_1_humidity: 282.4534 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 31/100
10/10 [==============================] - ETA: 0s - loss: 26493.6191 - RMSE_step_t_humidity: 281.7704 - RMSE_step_t_1_humidity: 281.7592 - RMSE_step_t_wind_dir: 3.8561 - RMSE_step_t_1_wind_dir: 4.0386 - RMSE_step_t_wind_speed: 2.7876 - RMSE_step_t_1_wind_speed: 2.9954
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 439ms/step - loss: 26493.6191 - RMSE_step_t_humidity: 281.7704 - RMSE_step_t_1_humidity: 281.7592 - RMSE_step_t_wind_dir: 3.8561 - RMSE_step_t_1_wind_dir: 4.0386 - RMSE_step_t_wind_speed: 2.7876 - RMSE_step_t_1_wind_speed: 2.9954 - val_loss: 26539.2480 - val_RMSE_step_t_humidity: 282.5796 - val_RMSE_step_t_1_humidity: 282.4534 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6041 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 32/100
10/10 [==============================] - ETA: 0s - loss: 26493.2422 - RMSE_step_t_humidity: 281.9027 - RMSE_step_t_1_humidity: 281.8525 - RMSE_step_t_wind_dir: 3.8930 - RMSE_step_t_1_wind_dir: 4.0633 - RMSE_step_t_wind_speed: 2.7873 - RMSE_step_t_1_wind_speed: 3.0328
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 434ms/step - loss: 26493.2422 - RMSE_step_t_humidity: 281.9027 - RMSE_step_t_1_humidity: 281.8525 - RMSE_step_t_wind_dir: 3.8930 - RMSE_step_t_1_wind_dir: 4.0633 - RMSE_step_t_wind_speed: 2.7873 - RMSE_step_t_1_wind_speed: 3.0328 - val_loss: 26539.2402 - val_RMSE_step_t_humidity: 282.5796 - val_RMSE_step_t_1_humidity: 282.4534 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 33/100
10/10 [==============================] - ETA: 0s - loss: 26492.7441 - RMSE_step_t_humidity: 281.7426 - RMSE_step_t_1_humidity: 281.7032 - RMSE_step_t_wind_dir: 3.8519 - RMSE_step_t_1_wind_dir: 4.0180 - RMSE_step_t_wind_speed: 2.8089 - RMSE_step_t_1_wind_speed: 3.0145
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 488ms/step - loss: 26492.7441 - RMSE_step_t_humidity: 281.7426 - RMSE_step_t_1_humidity: 281.7032 - RMSE_step_t_wind_dir: 3.8519 - RMSE_step_t_1_wind_dir: 4.0180 - RMSE_step_t_wind_speed: 2.8089 - RMSE_step_t_1_wind_speed: 3.0145 - val_loss: 26539.2344 - val_RMSE_step_t_humidity: 282.5795 - val_RMSE_step_t_1_humidity: 282.4534 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 34/100
10/10 [==============================] - ETA: 0s - loss: 26493.4082 - RMSE_step_t_humidity: 281.8075 - RMSE_step_t_1_humidity: 281.7917 - RMSE_step_t_wind_dir: 3.8282 - RMSE_step_t_1_wind_dir: 4.0186 - RMSE_step_t_wind_speed: 2.7642 - RMSE_step_t_1_wind_speed: 2.9868
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 425ms/step - loss: 26493.4082 - RMSE_step_t_humidity: 281.8075 - RMSE_step_t_1_humidity: 281.7917 - RMSE_step_t_wind_dir: 3.8282 - RMSE_step_t_1_wind_dir: 4.0186 - RMSE_step_t_wind_speed: 2.7642 - RMSE_step_t_1_wind_speed: 2.9868 - val_loss: 26539.2285 - val_RMSE_step_t_humidity: 282.5795 - val_RMSE_step_t_1_humidity: 282.4533 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 35/100
10/10 [==============================] - ETA: 0s - loss: 26492.6660 - RMSE_step_t_humidity: 281.9272 - RMSE_step_t_1_humidity: 281.8927 - RMSE_step_t_wind_dir: 3.8664 - RMSE_step_t_1_wind_dir: 4.0191 - RMSE_step_t_wind_speed: 2.7999 - RMSE_step_t_1_wind_speed: 2.9968
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 456ms/step - loss: 26492.6660 - RMSE_step_t_humidity: 281.9272 - RMSE_step_t_1_humidity: 281.8927 - RMSE_step_t_wind_dir: 3.8664 - RMSE_step_t_1_wind_dir: 4.0191 - RMSE_step_t_wind_speed: 2.7999 - RMSE_step_t_1_wind_speed: 2.9968 - val_loss: 26539.2207 - val_RMSE_step_t_humidity: 282.5795 - val_RMSE_step_t_1_humidity: 282.4533 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5780
Epoch 36/100
10/10 [==============================] - ETA: 0s - loss: 26493.3906 - RMSE_step_t_humidity: 281.7724 - RMSE_step_t_1_humidity: 281.8512 - RMSE_step_t_wind_dir: 3.8222 - RMSE_step_t_1_wind_dir: 4.0013 - RMSE_step_t_wind_speed: 2.7565 - RMSE_step_t_1_wind_speed: 2.9710
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 438ms/step - loss: 26493.3906 - RMSE_step_t_humidity: 281.7724 - RMSE_step_t_1_humidity: 281.8512 - RMSE_step_t_wind_dir: 3.8222 - RMSE_step_t_1_wind_dir: 4.0013 - RMSE_step_t_wind_speed: 2.7565 - RMSE_step_t_1_wind_speed: 2.9710 - val_loss: 26539.2188 - val_RMSE_step_t_humidity: 282.5794 - val_RMSE_step_t_1_humidity: 282.4533 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 37/100
10/10 [==============================] - ETA: 0s - loss: 26492.1738 - RMSE_step_t_humidity: 281.8729 - RMSE_step_t_1_humidity: 281.8185 - RMSE_step_t_wind_dir: 3.8389 - RMSE_step_t_1_wind_dir: 4.0424 - RMSE_step_t_wind_speed: 2.7604 - RMSE_step_t_1_wind_speed: 3.0145
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 436ms/step - loss: 26492.1738 - RMSE_step_t_humidity: 281.8729 - RMSE_step_t_1_humidity: 281.8185 - RMSE_step_t_wind_dir: 3.8389 - RMSE_step_t_1_wind_dir: 4.0424 - RMSE_step_t_wind_speed: 2.7604 - RMSE_step_t_1_wind_speed: 3.0145 - val_loss: 26539.2051 - val_RMSE_step_t_humidity: 282.5794 - val_RMSE_step_t_1_humidity: 282.4532 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 38/100
10/10 [==============================] - ETA: 0s - loss: 26492.9023 - RMSE_step_t_humidity: 281.8488 - RMSE_step_t_1_humidity: 281.8094 - RMSE_step_t_wind_dir: 3.8405 - RMSE_step_t_1_wind_dir: 3.9641 - RMSE_step_t_wind_speed: 2.7622 - RMSE_step_t_1_wind_speed: 2.9616
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 433ms/step - loss: 26492.9023 - RMSE_step_t_humidity: 281.8488 - RMSE_step_t_1_humidity: 281.8094 - RMSE_step_t_wind_dir: 3.8405 - RMSE_step_t_1_wind_dir: 3.9641 - RMSE_step_t_wind_speed: 2.7622 - RMSE_step_t_1_wind_speed: 2.9616 - val_loss: 26539.2031 - val_RMSE_step_t_humidity: 282.5793 - val_RMSE_step_t_1_humidity: 282.4532 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 39/100
10/10 [==============================] - ETA: 0s - loss: 26493.1504 - RMSE_step_t_humidity: 281.8865 - RMSE_step_t_1_humidity: 281.9710 - RMSE_step_t_wind_dir: 3.8756 - RMSE_step_t_1_wind_dir: 4.0634 - RMSE_step_t_wind_speed: 2.7816 - RMSE_step_t_1_wind_speed: 3.0115
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 433ms/step - loss: 26493.1504 - RMSE_step_t_humidity: 281.8865 - RMSE_step_t_1_humidity: 281.9710 - RMSE_step_t_wind_dir: 3.8756 - RMSE_step_t_1_wind_dir: 4.0634 - RMSE_step_t_wind_speed: 2.7816 - RMSE_step_t_1_wind_speed: 3.0115 - val_loss: 26539.1973 - val_RMSE_step_t_humidity: 282.5793 - val_RMSE_step_t_1_humidity: 282.4531 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 40/100
10/10 [==============================] - ETA: 0s - loss: 26491.7207 - RMSE_step_t_humidity: 281.8229 - RMSE_step_t_1_humidity: 281.7651 - RMSE_step_t_wind_dir: 3.8310 - RMSE_step_t_1_wind_dir: 3.9991 - RMSE_step_t_wind_speed: 2.7864 - RMSE_step_t_1_wind_speed: 2.9920
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 456ms/step - loss: 26491.7207 - RMSE_step_t_humidity: 281.8229 - RMSE_step_t_1_humidity: 281.7651 - RMSE_step_t_wind_dir: 3.8310 - RMSE_step_t_1_wind_dir: 3.9991 - RMSE_step_t_wind_speed: 2.7864 - RMSE_step_t_1_wind_speed: 2.9920 - val_loss: 26539.1895 - val_RMSE_step_t_humidity: 282.5793 - val_RMSE_step_t_1_humidity: 282.4531 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 41/100
10/10 [==============================] - 4s 413ms/step - loss: 26493.5000 - RMSE_step_t_humidity: 281.8111 - RMSE_step_t_1_humidity: 281.7690 - RMSE_step_t_wind_dir: 3.8184 - RMSE_step_t_1_wind_dir: 4.0120 - RMSE_step_t_wind_speed: 2.7776 - RMSE_step_t_1_wind_speed: 3.0285 - val_loss: 26539.1855 - val_RMSE_step_t_humidity: 282.5793 - val_RMSE_step_t_1_humidity: 282.4531 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 42/100
10/10 [==============================] - ETA: 0s - loss: 26493.3730 - RMSE_step_t_humidity: 281.7844 - RMSE_step_t_1_humidity: 281.7823 - RMSE_step_t_wind_dir: 3.8798 - RMSE_step_t_1_wind_dir: 4.0376 - RMSE_step_t_wind_speed: 2.7705 - RMSE_step_t_1_wind_speed: 2.9893
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 443ms/step - loss: 26493.3730 - RMSE_step_t_humidity: 281.7844 - RMSE_step_t_1_humidity: 281.7823 - RMSE_step_t_wind_dir: 3.8798 - RMSE_step_t_1_wind_dir: 4.0376 - RMSE_step_t_wind_speed: 2.7705 - RMSE_step_t_1_wind_speed: 2.9893 - val_loss: 26539.1797 - val_RMSE_step_t_humidity: 282.5793 - val_RMSE_step_t_1_humidity: 282.4530 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 43/100
10/10 [==============================] - ETA: 0s - loss: 26492.0195 - RMSE_step_t_humidity: 281.9784 - RMSE_step_t_1_humidity: 281.9961 - RMSE_step_t_wind_dir: 3.9092 - RMSE_step_t_1_wind_dir: 4.0731 - RMSE_step_t_wind_speed: 2.7869 - RMSE_step_t_1_wind_speed: 3.0053
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 442ms/step - loss: 26492.0195 - RMSE_step_t_humidity: 281.9784 - RMSE_step_t_1_humidity: 281.9961 - RMSE_step_t_wind_dir: 3.9092 - RMSE_step_t_1_wind_dir: 4.0731 - RMSE_step_t_wind_speed: 2.7869 - RMSE_step_t_1_wind_speed: 3.0053 - val_loss: 26539.1738 - val_RMSE_step_t_humidity: 282.5792 - val_RMSE_step_t_1_humidity: 282.4530 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 44/100
10/10 [==============================] - ETA: 0s - loss: 26490.8125 - RMSE_step_t_humidity: 281.9436 - RMSE_step_t_1_humidity: 281.8961 - RMSE_step_t_wind_dir: 3.8785 - RMSE_step_t_1_wind_dir: 4.0348 - RMSE_step_t_wind_speed: 2.7985 - RMSE_step_t_1_wind_speed: 3.0026
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 429ms/step - loss: 26490.8125 - RMSE_step_t_humidity: 281.9436 - RMSE_step_t_1_humidity: 281.8961 - RMSE_step_t_wind_dir: 3.8785 - RMSE_step_t_1_wind_dir: 4.0348 - RMSE_step_t_wind_speed: 2.7985 - RMSE_step_t_1_wind_speed: 3.0026 - val_loss: 26539.1660 - val_RMSE_step_t_humidity: 282.5792 - val_RMSE_step_t_1_humidity: 282.4530 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 45/100
10/10 [==============================] - ETA: 0s - loss: 26492.1641 - RMSE_step_t_humidity: 281.7934 - RMSE_step_t_1_humidity: 281.7310 - RMSE_step_t_wind_dir: 3.8233 - RMSE_step_t_1_wind_dir: 4.0066 - RMSE_step_t_wind_speed: 2.7917 - RMSE_step_t_1_wind_speed: 3.0059
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 6s 623ms/step - loss: 26492.1641 - RMSE_step_t_humidity: 281.7934 - RMSE_step_t_1_humidity: 281.7310 - RMSE_step_t_wind_dir: 3.8233 - RMSE_step_t_1_wind_dir: 4.0066 - RMSE_step_t_wind_speed: 2.7917 - RMSE_step_t_1_wind_speed: 3.0059 - val_loss: 26539.1582 - val_RMSE_step_t_humidity: 282.5791 - val_RMSE_step_t_1_humidity: 282.4529 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 46/100
10/10 [==============================] - ETA: 0s - loss: 26493.1484 - RMSE_step_t_humidity: 282.0298 - RMSE_step_t_1_humidity: 282.0216 - RMSE_step_t_wind_dir: 3.9158 - RMSE_step_t_1_wind_dir: 4.0896 - RMSE_step_t_wind_speed: 2.8105 - RMSE_step_t_1_wind_speed: 3.0318
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 483ms/step - loss: 26493.1484 - RMSE_step_t_humidity: 282.0298 - RMSE_step_t_1_humidity: 282.0216 - RMSE_step_t_wind_dir: 3.9158 - RMSE_step_t_1_wind_dir: 4.0896 - RMSE_step_t_wind_speed: 2.8105 - RMSE_step_t_1_wind_speed: 3.0318 - val_loss: 26539.1504 - val_RMSE_step_t_humidity: 282.5791 - val_RMSE_step_t_1_humidity: 282.4529 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 47/100
10/10 [==============================] - ETA: 0s - loss: 26492.7832 - RMSE_step_t_humidity: 281.7691 - RMSE_step_t_1_humidity: 281.7201 - RMSE_step_t_wind_dir: 3.7861 - RMSE_step_t_1_wind_dir: 3.9405 - RMSE_step_t_wind_speed: 2.7670 - RMSE_step_t_1_wind_speed: 2.9526
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 455ms/step - loss: 26492.7832 - RMSE_step_t_humidity: 281.7691 - RMSE_step_t_1_humidity: 281.7201 - RMSE_step_t_wind_dir: 3.7861 - RMSE_step_t_1_wind_dir: 3.9405 - RMSE_step_t_wind_speed: 2.7670 - RMSE_step_t_1_wind_speed: 2.9526 - val_loss: 26539.1465 - val_RMSE_step_t_humidity: 282.5791 - val_RMSE_step_t_1_humidity: 282.4529 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6040 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 48/100
10/10 [==============================] - ETA: 0s - loss: 26491.8496 - RMSE_step_t_humidity: 281.9163 - RMSE_step_t_1_humidity: 281.8590 - RMSE_step_t_wind_dir: 3.8632 - RMSE_step_t_1_wind_dir: 4.0198 - RMSE_step_t_wind_speed: 2.8131 - RMSE_step_t_1_wind_speed: 3.0050
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 407ms/step - loss: 26491.8496 - RMSE_step_t_humidity: 281.9163 - RMSE_step_t_1_humidity: 281.8590 - RMSE_step_t_wind_dir: 3.8632 - RMSE_step_t_1_wind_dir: 4.0198 - RMSE_step_t_wind_speed: 2.8131 - RMSE_step_t_1_wind_speed: 3.0050 - val_loss: 26539.1406 - val_RMSE_step_t_humidity: 282.5790 - val_RMSE_step_t_1_humidity: 282.4529 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 49/100
10/10 [==============================] - ETA: 0s - loss: 26492.5488 - RMSE_step_t_humidity: 281.9085 - RMSE_step_t_1_humidity: 281.8460 - RMSE_step_t_wind_dir: 3.8173 - RMSE_step_t_1_wind_dir: 3.9596 - RMSE_step_t_wind_speed: 2.7898 - RMSE_step_t_1_wind_speed: 2.9808
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 448ms/step - loss: 26492.5488 - RMSE_step_t_humidity: 281.9085 - RMSE_step_t_1_humidity: 281.8460 - RMSE_step_t_wind_dir: 3.8173 - RMSE_step_t_1_wind_dir: 3.9596 - RMSE_step_t_wind_speed: 2.7898 - RMSE_step_t_1_wind_speed: 2.9808 - val_loss: 26539.1348 - val_RMSE_step_t_humidity: 282.5790 - val_RMSE_step_t_1_humidity: 282.4528 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 50/100
10/10 [==============================] - ETA: 0s - loss: 26492.9062 - RMSE_step_t_humidity: 281.7495 - RMSE_step_t_1_humidity: 281.7119 - RMSE_step_t_wind_dir: 3.8187 - RMSE_step_t_1_wind_dir: 3.9722 - RMSE_step_t_wind_speed: 2.7445 - RMSE_step_t_1_wind_speed: 2.9539
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - 4s 429ms/step - loss: 26492.9062 - RMSE_step_t_humidity: 281.7495 - RMSE_step_t_1_humidity: 281.7119 - RMSE_step_t_wind_dir: 3.8187 - RMSE_step_t_1_wind_dir: 3.9722 - RMSE_step_t_wind_speed: 2.7445 - RMSE_step_t_1_wind_speed: 2.9539 - val_loss: 26539.1270 - val_RMSE_step_t_humidity: 282.5789 - val_RMSE_step_t_1_humidity: 282.4528 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 51/100
10/10 [==============================] - ETA: 0s - loss: 26491.6367 - RMSE_step_t_humidity: 282.0508 - RMSE_step_t_1_humidity: 282.0302 - RMSE_step_t_wind_dir: 3.8852 - RMSE_step_t_1_wind_dir: 4.0715 - RMSE_step_t_wind_speed: 2.8115 - RMSE_step_t_1_wind_speed: 3.0496
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
 9/10 [==========================>...] - ETA: 0s - loss: 26498.4219 - RMSE_step_t_humidity: 281.8748 - RMSE_step_t_1_humidity: 281.8545 - RMSE_step_t_wind_dir: 3.8726 - RMSE_step_t_1_wind_dir: 4.0450 - RMSE_step_t_wind_speed: 2.7970 - RMSE_step_t_1_wind_speed: 3.02090496 - val_loss: 26539.1250 - val_RMSE_step_t_humidity: 282.5789 - val_RMSE_step_t_1_humidity: 282.4528 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5779
Epoch 52/100
 9/10 [==========================>...] - ETA: 0s - loss: 26498.4219 - RMSE_step_t_humidity: 281.8748 - RMSE_step_t_1_humidity: 281.8545 - RMSE_step_t_wind_dir: 3.8726 - RMSE_step_t_1_wind_dir: 4.0450 - RMSE_step_t_wind_speed: 2.7970 - RMSE_step_t_1_wind_speed: 3.02090496 - val_loss: 26539.1250 - val_RMSE_step_t_humidity: 282.5789 - val_RMSE_step_t_1_humidity: 282.4528 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5779
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.9883 - RMSE_step_t_humidity: 281.8389 - RMSE_step_t_1_humidity: 281.8147 - RMSE_step_t_wind_dir: 3.9316 - RMSE_step_t_1_wind_dir: 4.0971 - RMSE_step_t_wind_speed: 2.8080 - RMSE_step_t_1_wind_speed: 3.02269712 - val_loss: 26539.1113 - val_RMSE_step_t_humidity: 282.5789 - val_RMSE_step_t_1_humidity: 282.4527 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 53/100
10/10 [==============================] - ETA: 0s - loss: 26491.9883 - RMSE_step_t_humidity: 281.8389 - RMSE_step_t_1_humidity: 281.8147 - RMSE_step_t_wind_dir: 3.9316 - RMSE_step_t_1_wind_dir: 4.0971 - RMSE_step_t_wind_speed: 2.8080 - RMSE_step_t_1_wind_speed: 3.02269712 - val_loss: 26539.1113 - val_RMSE_step_t_humidity: 282.5789 - val_RMSE_step_t_1_humidity: 282.4527 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9015 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.2480 - RMSE_step_t_humidity: 281.7533 - RMSE_step_t_1_humidity: 281.7069 - RMSE_step_t_wind_dir: 3.8171 - RMSE_step_t_1_wind_dir: 3.9765 - RMSE_step_t_wind_speed: 2.7336 - RMSE_step_t_1_wind_speed: 2.94803.0226 - val_loss: 26539.1094 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4527 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 54/100
10/10 [==============================] - ETA: 0s - loss: 26491.2480 - RMSE_step_t_humidity: 281.7533 - RMSE_step_t_1_humidity: 281.7069 - RMSE_step_t_wind_dir: 3.8171 - RMSE_step_t_1_wind_dir: 3.9765 - RMSE_step_t_wind_speed: 2.7336 - RMSE_step_t_1_wind_speed: 2.94803.0226 - val_loss: 26539.1094 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4527 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.9277 - RMSE_step_t_humidity: 281.9211 - RMSE_step_t_1_humidity: 281.8840 - RMSE_step_t_wind_dir: 3.8273 - RMSE_step_t_1_wind_dir: 3.9894 - RMSE_step_t_wind_speed: 2.8102 - RMSE_step_t_1_wind_speed: 3.04152.9480 - val_loss: 26539.1016 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4526 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 55/100
10/10 [==============================] - ETA: 0s - loss: 26492.6699 - RMSE_step_t_humidity: 281.8192 - RMSE_step_t_1_humidity: 281.7905 - RMSE_step_t_wind_dir: 3.8683 - RMSE_step_t_1_wind_dir: 4.0434 - RMSE_step_t_wind_speed: 2.7664 - RMSE_step_t_1_wind_speed: 2.98553.0415 - val_loss: 26539.0957 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4526 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 56/100
10/10 [==============================] - ETA: 0s - loss: 26492.6699 - RMSE_step_t_humidity: 281.8192 - RMSE_step_t_1_humidity: 281.7905 - RMSE_step_t_wind_dir: 3.8683 - RMSE_step_t_1_wind_dir: 4.0434 - RMSE_step_t_wind_speed: 2.7664 - RMSE_step_t_1_wind_speed: 2.98553.0415 - val_loss: 26539.0957 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4526 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.2207 - RMSE_step_t_humidity: 281.9138 - RMSE_step_t_1_humidity: 281.8742 - RMSE_step_t_wind_dir: 3.8561 - RMSE_step_t_1_wind_dir: 4.0315 - RMSE_step_t_wind_speed: 2.8122 - RMSE_step_t_1_wind_speed: 3.04542.9855 - val_loss: 26539.0918 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4526 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 57/100
10/10 [==============================] - ETA: 0s - loss: 26492.2207 - RMSE_step_t_humidity: 281.9138 - RMSE_step_t_1_humidity: 281.8742 - RMSE_step_t_wind_dir: 3.8561 - RMSE_step_t_1_wind_dir: 4.0315 - RMSE_step_t_wind_speed: 2.8122 - RMSE_step_t_1_wind_speed: 3.04542.9855 - val_loss: 26539.0918 - val_RMSE_step_t_humidity: 282.5788 - val_RMSE_step_t_1_humidity: 282.4526 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.1426 - RMSE_step_t_humidity: 281.9236 - RMSE_step_t_1_humidity: 281.9189 - RMSE_step_t_wind_dir: 3.9200 - RMSE_step_t_1_wind_dir: 4.0497 - RMSE_step_t_wind_speed: 2.7980 - RMSE_step_t_1_wind_speed: 2.98403.0454 - val_loss: 26539.0840 - val_RMSE_step_t_humidity: 282.5787 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 58/100
10/10 [==============================] - ETA: 0s - loss: 26491.4980 - RMSE_step_t_humidity: 281.8251 - RMSE_step_t_1_humidity: 281.7946 - RMSE_step_t_wind_dir: 3.8971 - RMSE_step_t_1_wind_dir: 4.0455 - RMSE_step_t_wind_speed: 2.7978 - RMSE_step_t_1_wind_speed: 2.99942.9840 - val_loss: 26539.0801 - val_RMSE_step_t_humidity: 282.5787 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 59/100
10/10 [==============================] - ETA: 0s - loss: 26491.4980 - RMSE_step_t_humidity: 281.8251 - RMSE_step_t_1_humidity: 281.7946 - RMSE_step_t_wind_dir: 3.8971 - RMSE_step_t_1_wind_dir: 4.0455 - RMSE_step_t_wind_speed: 2.7978 - RMSE_step_t_1_wind_speed: 2.99942.9840 - val_loss: 26539.0801 - val_RMSE_step_t_humidity: 282.5787 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.0000 - RMSE_step_t_humidity: 281.9128 - RMSE_step_t_1_humidity: 281.8818 - RMSE_step_t_wind_dir: 3.8808 - RMSE_step_t_1_wind_dir: 4.0449 - RMSE_step_t_wind_speed: 2.7806 - RMSE_step_t_1_wind_speed: 3.00362.9994 - val_loss: 26539.0723 - val_RMSE_step_t_humidity: 282.5786 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 60/100
10/10 [==============================] - ETA: 0s - loss: 26492.0000 - RMSE_step_t_humidity: 281.9128 - RMSE_step_t_1_humidity: 281.8818 - RMSE_step_t_wind_dir: 3.8808 - RMSE_step_t_1_wind_dir: 4.0449 - RMSE_step_t_wind_speed: 2.7806 - RMSE_step_t_1_wind_speed: 3.00362.9994 - val_loss: 26539.0723 - val_RMSE_step_t_humidity: 282.5786 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.0137 - RMSE_step_t_humidity: 281.8085 - RMSE_step_t_1_humidity: 281.7487 - RMSE_step_t_wind_dir: 3.8261 - RMSE_step_t_1_wind_dir: 3.9971 - RMSE_step_t_wind_speed: 2.7810 - RMSE_step_t_1_wind_speed: 2.98083.0036 - val_loss: 26539.0645 - val_RMSE_step_t_humidity: 282.5786 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 61/100
10/10 [==============================] - ETA: 0s - loss: 26492.0137 - RMSE_step_t_humidity: 281.8085 - RMSE_step_t_1_humidity: 281.7487 - RMSE_step_t_wind_dir: 3.8261 - RMSE_step_t_1_wind_dir: 3.9971 - RMSE_step_t_wind_speed: 2.7810 - RMSE_step_t_1_wind_speed: 2.98083.0036 - val_loss: 26539.0645 - val_RMSE_step_t_humidity: 282.5786 - val_RMSE_step_t_1_humidity: 282.4525 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
 9/10 [==========================>...] - ETA: 0s - loss: 26494.9277 - RMSE_step_t_humidity: 281.8634 - RMSE_step_t_1_humidity: 281.8287 - RMSE_step_t_wind_dir: 3.8753 - RMSE_step_t_1_wind_dir: 4.0508 - RMSE_step_t_wind_speed: 2.7814 - RMSE_step_t_1_wind_speed: 3.01032.9808 - val_loss: 26539.0605 - val_RMSE_step_t_humidity: 282.5786 - val_RMSE_step_t_1_humidity: 282.4524 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 62/100
 9/10 [==========================>...] - ETA: 0s - loss: 26494.9277 - RMSE_step_t_humidity: 281.8634 - RMSE_step_t_1_humidity: 281.8287 - RMSE_step_t_wind_dir: 3.8753 - RMSE_step_t_1_wind_dir: 4.0508 - RMSE_step_t_wind_speed: 2.7814 - RMSE_step_t_1_wind_speed: 3.01032.9808 - val_loss: 26539.0605 - val_RMSE_step_t_humidity: 282.5786 - val_RMSE_step_t_1_humidity: 282.4524 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.2266 - RMSE_step_t_humidity: 281.8282 - RMSE_step_t_1_humidity: 281.7575 - RMSE_step_t_wind_dir: 3.8892 - RMSE_step_t_1_wind_dir: 4.0613 - RMSE_step_t_wind_speed: 2.7978 - RMSE_step_t_1_wind_speed: 3.01982.9805 - val_loss: 26539.0488 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4524 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 63/100
10/10 [==============================] - ETA: 0s - loss: 26491.2266 - RMSE_step_t_humidity: 281.8282 - RMSE_step_t_1_humidity: 281.7575 - RMSE_step_t_wind_dir: 3.8892 - RMSE_step_t_1_wind_dir: 4.0613 - RMSE_step_t_wind_speed: 2.7978 - RMSE_step_t_1_wind_speed: 3.01982.9805 - val_loss: 26539.0488 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4524 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.5781 - RMSE_step_t_humidity: 281.8666 - RMSE_step_t_1_humidity: 281.8371 - RMSE_step_t_wind_dir: 3.9015 - RMSE_step_t_1_wind_dir: 4.0755 - RMSE_step_t_wind_speed: 2.8065 - RMSE_step_t_1_wind_speed: 3.01393.0198 - val_loss: 26539.0469 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 64/100
10/10 [==============================] - ETA: 0s - loss: 26491.5781 - RMSE_step_t_humidity: 281.8666 - RMSE_step_t_1_humidity: 281.8371 - RMSE_step_t_wind_dir: 3.9015 - RMSE_step_t_1_wind_dir: 4.0755 - RMSE_step_t_wind_speed: 2.8065 - RMSE_step_t_1_wind_speed: 3.01393.0198 - val_loss: 26539.0469 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.6523 - RMSE_step_t_humidity: 281.8244 - RMSE_step_t_1_humidity: 281.8161 - RMSE_step_t_wind_dir: 3.8660 - RMSE_step_t_1_wind_dir: 4.0487 - RMSE_step_t_wind_speed: 2.7873 - RMSE_step_t_1_wind_speed: 3.02143.0139 - val_loss: 26539.0391 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 65/100
10/10 [==============================] - ETA: 0s - loss: 26492.6523 - RMSE_step_t_humidity: 281.8244 - RMSE_step_t_1_humidity: 281.8161 - RMSE_step_t_wind_dir: 3.8660 - RMSE_step_t_1_wind_dir: 4.0487 - RMSE_step_t_wind_speed: 2.7873 - RMSE_step_t_1_wind_speed: 3.02143.0139 - val_loss: 26539.0391 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6039 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.5996 - RMSE_step_t_humidity: 281.7935 - RMSE_step_t_1_humidity: 281.7407 - RMSE_step_t_wind_dir: 3.7910 - RMSE_step_t_1_wind_dir: 3.9658 - RMSE_step_t_wind_speed: 2.7146 - RMSE_step_t_1_wind_speed: 2.94143.0214 - val_loss: 26539.0332 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 66/100
10/10 [==============================] - ETA: 0s - loss: 26490.5996 - RMSE_step_t_humidity: 281.7935 - RMSE_step_t_1_humidity: 281.7407 - RMSE_step_t_wind_dir: 3.7910 - RMSE_step_t_1_wind_dir: 3.9658 - RMSE_step_t_wind_speed: 2.7146 - RMSE_step_t_1_wind_speed: 2.94143.0214 - val_loss: 26539.0332 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.5137 - RMSE_step_t_humidity: 281.8577 - RMSE_step_t_1_humidity: 281.8065 - RMSE_step_t_wind_dir: 3.8436 - RMSE_step_t_1_wind_dir: 4.0326 - RMSE_step_t_wind_speed: 2.7564 - RMSE_step_t_1_wind_speed: 2.99702.9414 - val_loss: 26539.0293 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 67/100
10/10 [==============================] - ETA: 0s - loss: 26490.5137 - RMSE_step_t_humidity: 281.8577 - RMSE_step_t_1_humidity: 281.8065 - RMSE_step_t_wind_dir: 3.8436 - RMSE_step_t_1_wind_dir: 4.0326 - RMSE_step_t_wind_speed: 2.7564 - RMSE_step_t_1_wind_speed: 2.99702.9414 - val_loss: 26539.0293 - val_RMSE_step_t_humidity: 282.5785 - val_RMSE_step_t_1_humidity: 282.4523 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5778
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26489.7520 - RMSE_step_t_humidity: 281.8437 - RMSE_step_t_1_humidity: 281.7842 - RMSE_step_t_wind_dir: 3.8162 - RMSE_step_t_1_wind_dir: 3.9231 - RMSE_step_t_wind_speed: 2.7979 - RMSE_step_t_1_wind_speed: 2.94572.9970 - val_loss: 26539.0215 - val_RMSE_step_t_humidity: 282.5784 - val_RMSE_step_t_1_humidity: 282.4522 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5778
Epoch 68/100
10/10 [==============================] - ETA: 0s - loss: 26490.9453 - RMSE_step_t_humidity: 281.8374 - RMSE_step_t_1_humidity: 281.8224 - RMSE_step_t_wind_dir: 3.8245 - RMSE_step_t_1_wind_dir: 3.9839 - RMSE_step_t_wind_speed: 2.7431 - RMSE_step_t_1_wind_speed: 2.94932.9457 - val_loss: 26539.0156 - val_RMSE_step_t_humidity: 282.5784 - val_RMSE_step_t_1_humidity: 282.4522 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 69/100
10/10 [==============================] - ETA: 0s - loss: 26490.9453 - RMSE_step_t_humidity: 281.8374 - RMSE_step_t_1_humidity: 281.8224 - RMSE_step_t_wind_dir: 3.8245 - RMSE_step_t_1_wind_dir: 3.9839 - RMSE_step_t_wind_speed: 2.7431 - RMSE_step_t_1_wind_speed: 2.94932.9457 - val_loss: 26539.0156 - val_RMSE_step_t_humidity: 282.5784 - val_RMSE_step_t_1_humidity: 282.4522 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.8477 - RMSE_step_t_humidity: 281.8111 - RMSE_step_t_1_humidity: 281.7949 - RMSE_step_t_wind_dir: 3.8715 - RMSE_step_t_1_wind_dir: 4.0471 - RMSE_step_t_wind_speed: 2.7446 - RMSE_step_t_1_wind_speed: 2.96252.9493 - val_loss: 26539.0020 - val_RMSE_step_t_humidity: 282.5783 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 70/100
10/10 [==============================] - ETA: 0s - loss: 26491.8477 - RMSE_step_t_humidity: 281.8111 - RMSE_step_t_1_humidity: 281.7949 - RMSE_step_t_wind_dir: 3.8715 - RMSE_step_t_1_wind_dir: 4.0471 - RMSE_step_t_wind_speed: 2.7446 - RMSE_step_t_1_wind_speed: 2.96252.9493 - val_loss: 26539.0020 - val_RMSE_step_t_humidity: 282.5783 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.0137 - RMSE_step_t_humidity: 281.6722 - RMSE_step_t_1_humidity: 281.6407 - RMSE_step_t_wind_dir: 3.8387 - RMSE_step_t_1_wind_dir: 4.0227 - RMSE_step_t_wind_speed: 2.7526 - RMSE_step_t_1_wind_speed: 2.98922.9625 - val_loss: 26539.0000 - val_RMSE_step_t_humidity: 282.5783 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 71/100
10/10 [==============================] - ETA: 0s - loss: 26492.0137 - RMSE_step_t_humidity: 281.6722 - RMSE_step_t_1_humidity: 281.6407 - RMSE_step_t_wind_dir: 3.8387 - RMSE_step_t_1_wind_dir: 4.0227 - RMSE_step_t_wind_speed: 2.7526 - RMSE_step_t_1_wind_speed: 2.98922.9625 - val_loss: 26539.0000 - val_RMSE_step_t_humidity: 282.5783 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.6875 - RMSE_step_t_humidity: 281.7804 - RMSE_step_t_1_humidity: 281.7583 - RMSE_step_t_wind_dir: 3.8455 - RMSE_step_t_1_wind_dir: 4.0314 - RMSE_step_t_wind_speed: 2.7840 - RMSE_step_t_1_wind_speed: 2.99462.9892 - val_loss: 26538.9922 - val_RMSE_step_t_humidity: 282.5783 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 72/100
10/10 [==============================] - ETA: 0s - loss: 26491.6875 - RMSE_step_t_humidity: 281.7804 - RMSE_step_t_1_humidity: 281.7583 - RMSE_step_t_wind_dir: 3.8455 - RMSE_step_t_1_wind_dir: 4.0314 - RMSE_step_t_wind_speed: 2.7840 - RMSE_step_t_1_wind_speed: 2.99462.9892 - val_loss: 26538.9922 - val_RMSE_step_t_humidity: 282.5783 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26489.9922 - RMSE_step_t_humidity: 281.5961 - RMSE_step_t_1_humidity: 281.6389 - RMSE_step_t_wind_dir: 3.8465 - RMSE_step_t_1_wind_dir: 4.0470 - RMSE_step_t_wind_speed: 2.7728 - RMSE_step_t_1_wind_speed: 2.99062.9946 - val_loss: 26538.9863 - val_RMSE_step_t_humidity: 282.5782 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 73/100
10/10 [==============================] - ETA: 0s - loss: 26489.9922 - RMSE_step_t_humidity: 281.5961 - RMSE_step_t_1_humidity: 281.6389 - RMSE_step_t_wind_dir: 3.8465 - RMSE_step_t_1_wind_dir: 4.0470 - RMSE_step_t_wind_speed: 2.7728 - RMSE_step_t_1_wind_speed: 2.99062.9946 - val_loss: 26538.9863 - val_RMSE_step_t_humidity: 282.5782 - val_RMSE_step_t_1_humidity: 282.4521 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.8418 - RMSE_step_t_humidity: 281.7964 - RMSE_step_t_1_humidity: 281.8015 - RMSE_step_t_wind_dir: 3.8880 - RMSE_step_t_1_wind_dir: 4.0797 - RMSE_step_t_wind_speed: 2.8045 - RMSE_step_t_1_wind_speed: 3.04012.9906 - val_loss: 26538.9844 - val_RMSE_step_t_humidity: 282.5782 - val_RMSE_step_t_1_humidity: 282.4520 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 74/100
10/10 [==============================] - ETA: 0s - loss: 26491.8418 - RMSE_step_t_humidity: 281.7964 - RMSE_step_t_1_humidity: 281.8015 - RMSE_step_t_wind_dir: 3.8880 - RMSE_step_t_1_wind_dir: 4.0797 - RMSE_step_t_wind_speed: 2.8045 - RMSE_step_t_1_wind_speed: 3.04012.9906 - val_loss: 26538.9844 - val_RMSE_step_t_humidity: 282.5782 - val_RMSE_step_t_1_humidity: 282.4520 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.9727 - RMSE_step_t_humidity: 281.9037 - RMSE_step_t_1_humidity: 281.8616 - RMSE_step_t_wind_dir: 3.9053 - RMSE_step_t_1_wind_dir: 4.0624 - RMSE_step_t_wind_speed: 2.7904 - RMSE_step_t_1_wind_speed: 2.99443.0401 - val_loss: 26538.9766 - val_RMSE_step_t_humidity: 282.5782 - val_RMSE_step_t_1_humidity: 282.4520 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 75/100
10/10 [==============================] - ETA: 0s - loss: 26491.7090 - RMSE_step_t_humidity: 281.7980 - RMSE_step_t_1_humidity: 281.7360 - RMSE_step_t_wind_dir: 3.8301 - RMSE_step_t_1_wind_dir: 3.9802 - RMSE_step_t_wind_speed: 2.7870 - RMSE_step_t_1_wind_speed: 2.99582.9944 - val_loss: 26538.9668 - val_RMSE_step_t_humidity: 282.5781 - val_RMSE_step_t_1_humidity: 282.4519 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 76/100
10/10 [==============================] - ETA: 0s - loss: 26491.7090 - RMSE_step_t_humidity: 281.7980 - RMSE_step_t_1_humidity: 281.7360 - RMSE_step_t_wind_dir: 3.8301 - RMSE_step_t_1_wind_dir: 3.9802 - RMSE_step_t_wind_speed: 2.7870 - RMSE_step_t_1_wind_speed: 2.99582.9944 - val_loss: 26538.9668 - val_RMSE_step_t_humidity: 282.5781 - val_RMSE_step_t_1_humidity: 282.4519 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.0117 - RMSE_step_t_humidity: 281.7281 - RMSE_step_t_1_humidity: 281.6707 - RMSE_step_t_wind_dir: 3.8224 - RMSE_step_t_1_wind_dir: 3.9793 - RMSE_step_t_wind_speed: 2.7597 - RMSE_step_t_1_wind_speed: 2.96792.9958 - val_loss: 26538.9609 - val_RMSE_step_t_humidity: 282.5781 - val_RMSE_step_t_1_humidity: 282.4519 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 77/100
10/10 [==============================] - ETA: 0s - loss: 26491.8848 - RMSE_step_t_humidity: 281.7929 - RMSE_step_t_1_humidity: 281.7792 - RMSE_step_t_wind_dir: 3.8521 - RMSE_step_t_1_wind_dir: 4.0482 - RMSE_step_t_wind_speed: 2.7565 - RMSE_step_t_1_wind_speed: 3.00722.9679 - val_loss: 26538.9551 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4519 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 78/100
10/10 [==============================] - ETA: 0s - loss: 26491.8848 - RMSE_step_t_humidity: 281.7929 - RMSE_step_t_1_humidity: 281.7792 - RMSE_step_t_wind_dir: 3.8521 - RMSE_step_t_1_wind_dir: 4.0482 - RMSE_step_t_wind_speed: 2.7565 - RMSE_step_t_1_wind_speed: 3.00722.9679 - val_loss: 26538.9551 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4519 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.3086 - RMSE_step_t_humidity: 281.6950 - RMSE_step_t_1_humidity: 281.6509 - RMSE_step_t_wind_dir: 3.7981 - RMSE_step_t_1_wind_dir: 3.9682 - RMSE_step_t_wind_speed: 2.7519 - RMSE_step_t_1_wind_speed: 2.99133.0072 - val_loss: 26538.9531 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 79/100
10/10 [==============================] - ETA: 0s - loss: 26491.3086 - RMSE_step_t_humidity: 281.6950 - RMSE_step_t_1_humidity: 281.6509 - RMSE_step_t_wind_dir: 3.7981 - RMSE_step_t_1_wind_dir: 3.9682 - RMSE_step_t_wind_speed: 2.7519 - RMSE_step_t_1_wind_speed: 2.99133.0072 - val_loss: 26538.9531 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26492.1621 - RMSE_step_t_humidity: 281.7827 - RMSE_step_t_1_humidity: 281.8120 - RMSE_step_t_wind_dir: 3.8431 - RMSE_step_t_1_wind_dir: 4.0355 - RMSE_step_t_wind_speed: 2.7613 - RMSE_step_t_1_wind_speed: 2.98442.9913 - val_loss: 26538.9453 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 80/100
10/10 [==============================] - ETA: 0s - loss: 26492.1621 - RMSE_step_t_humidity: 281.7827 - RMSE_step_t_1_humidity: 281.8120 - RMSE_step_t_wind_dir: 3.8431 - RMSE_step_t_1_wind_dir: 4.0355 - RMSE_step_t_wind_speed: 2.7613 - RMSE_step_t_1_wind_speed: 2.98442.9913 - val_loss: 26538.9453 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9125 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26489.8906 - RMSE_step_t_humidity: 281.8686 - RMSE_step_t_1_humidity: 281.8557 - RMSE_step_t_wind_dir: 3.9035 - RMSE_step_t_1_wind_dir: 4.0060 - RMSE_step_t_wind_speed: 2.7974 - RMSE_step_t_1_wind_speed: 2.96052.9844 - val_loss: 26538.9395 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 81/100
10/10 [==============================] - ETA: 0s - loss: 26489.8906 - RMSE_step_t_humidity: 281.8686 - RMSE_step_t_1_humidity: 281.8557 - RMSE_step_t_wind_dir: 3.9035 - RMSE_step_t_1_wind_dir: 4.0060 - RMSE_step_t_wind_speed: 2.7974 - RMSE_step_t_1_wind_speed: 2.96052.9844 - val_loss: 26538.9395 - val_RMSE_step_t_humidity: 282.5780 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6038 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.3281 - RMSE_step_t_humidity: 281.8804 - RMSE_step_t_1_humidity: 281.9148 - RMSE_step_t_wind_dir: 3.9002 - RMSE_step_t_1_wind_dir: 4.0722 - RMSE_step_t_wind_speed: 2.8079 - RMSE_step_t_1_wind_speed: 3.02532.9605 - val_loss: 26538.9316 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 82/100
10/10 [==============================] - ETA: 0s - loss: 26490.3281 - RMSE_step_t_humidity: 281.8804 - RMSE_step_t_1_humidity: 281.9148 - RMSE_step_t_wind_dir: 3.9002 - RMSE_step_t_1_wind_dir: 4.0722 - RMSE_step_t_wind_speed: 2.8079 - RMSE_step_t_1_wind_speed: 3.02532.9605 - val_loss: 26538.9316 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4518 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.6738 - RMSE_step_t_humidity: 281.8649 - RMSE_step_t_1_humidity: 281.8495 - RMSE_step_t_wind_dir: 3.8759 - RMSE_step_t_1_wind_dir: 4.0618 - RMSE_step_t_wind_speed: 2.7849 - RMSE_step_t_1_wind_speed: 3.02933.0253 - val_loss: 26538.9297 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4517 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 83/100
10/10 [==============================] - ETA: 0s - loss: 26491.6738 - RMSE_step_t_humidity: 281.8649 - RMSE_step_t_1_humidity: 281.8495 - RMSE_step_t_wind_dir: 3.8759 - RMSE_step_t_1_wind_dir: 4.0618 - RMSE_step_t_wind_speed: 2.7849 - RMSE_step_t_1_wind_speed: 3.02933.0253 - val_loss: 26538.9297 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4517 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.4922 - RMSE_step_t_humidity: 281.8544 - RMSE_step_t_1_humidity: 281.8936 - RMSE_step_t_wind_dir: 3.8765 - RMSE_step_t_1_wind_dir: 4.0720 - RMSE_step_t_wind_speed: 2.7702 - RMSE_step_t_1_wind_speed: 3.01033.0293 - val_loss: 26538.9160 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4517 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 84/100
10/10 [==============================] - ETA: 0s - loss: 26491.4922 - RMSE_step_t_humidity: 281.8544 - RMSE_step_t_1_humidity: 281.8936 - RMSE_step_t_wind_dir: 3.8765 - RMSE_step_t_1_wind_dir: 4.0720 - RMSE_step_t_wind_speed: 2.7702 - RMSE_step_t_1_wind_speed: 3.01033.0293 - val_loss: 26538.9160 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4517 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.2871 - RMSE_step_t_humidity: 281.8438 - RMSE_step_t_1_humidity: 281.7696 - RMSE_step_t_wind_dir: 3.8786 - RMSE_step_t_1_wind_dir: 4.0320 - RMSE_step_t_wind_speed: 2.7820 - RMSE_step_t_1_wind_speed: 2.98263.0103 - val_loss: 26538.9082 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4517 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
Epoch 85/100
10/10 [==============================] - ETA: 0s - loss: 26490.2871 - RMSE_step_t_humidity: 281.8438 - RMSE_step_t_1_humidity: 281.7696 - RMSE_step_t_wind_dir: 3.8786 - RMSE_step_t_1_wind_dir: 4.0320 - RMSE_step_t_wind_speed: 2.7820 - RMSE_step_t_1_wind_speed: 2.98263.0103 - val_loss: 26538.9082 - val_RMSE_step_t_humidity: 282.5779 - val_RMSE_step_t_1_humidity: 282.4517 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5777
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26489.4355 - RMSE_step_t_humidity: 281.8912 - RMSE_step_t_1_humidity: 281.8244 - RMSE_step_t_wind_dir: 3.8851 - RMSE_step_t_1_wind_dir: 4.0364 - RMSE_step_t_wind_speed: 2.8190 - RMSE_step_t_1_wind_speed: 3.01302.9826 - val_loss: 26538.9062 - val_RMSE_step_t_humidity: 282.5778 - val_RMSE_step_t_1_humidity: 282.4516 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 86/100
10/10 [==============================] - ETA: 0s - loss: 26489.4355 - RMSE_step_t_humidity: 281.8912 - RMSE_step_t_1_humidity: 281.8244 - RMSE_step_t_wind_dir: 3.8851 - RMSE_step_t_1_wind_dir: 4.0364 - RMSE_step_t_wind_speed: 2.8190 - RMSE_step_t_1_wind_speed: 3.01302.9826 - val_loss: 26538.9062 - val_RMSE_step_t_humidity: 282.5778 - val_RMSE_step_t_1_humidity: 282.4516 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.0996 - RMSE_step_t_humidity: 281.8388 - RMSE_step_t_1_humidity: 281.7790 - RMSE_step_t_wind_dir: 3.8239 - RMSE_step_t_1_wind_dir: 3.9898 - RMSE_step_t_wind_speed: 2.7842 - RMSE_step_t_1_wind_speed: 3.00523.0130 - val_loss: 26538.8984 - val_RMSE_step_t_humidity: 282.5778 - val_RMSE_step_t_1_humidity: 282.4516 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 87/100
10/10 [==============================] - ETA: 0s - loss: 26491.0996 - RMSE_step_t_humidity: 281.8388 - RMSE_step_t_1_humidity: 281.7790 - RMSE_step_t_wind_dir: 3.8239 - RMSE_step_t_1_wind_dir: 3.9898 - RMSE_step_t_wind_speed: 2.7842 - RMSE_step_t_1_wind_speed: 3.00523.0130 - val_loss: 26538.8984 - val_RMSE_step_t_humidity: 282.5778 - val_RMSE_step_t_1_humidity: 282.4516 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.7422 - RMSE_step_t_humidity: 281.8471 - RMSE_step_t_1_humidity: 281.8051 - RMSE_step_t_wind_dir: 3.8261 - RMSE_step_t_1_wind_dir: 3.9956 - RMSE_step_t_wind_speed: 2.7712 - RMSE_step_t_1_wind_speed: 2.97273.0052 - val_loss: 26538.8926 - val_RMSE_step_t_humidity: 282.5777 - val_RMSE_step_t_1_humidity: 282.4516 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 88/100
10/10 [==============================] - ETA: 0s - loss: 26490.7422 - RMSE_step_t_humidity: 281.8471 - RMSE_step_t_1_humidity: 281.8051 - RMSE_step_t_wind_dir: 3.8261 - RMSE_step_t_1_wind_dir: 3.9956 - RMSE_step_t_wind_speed: 2.7712 - RMSE_step_t_1_wind_speed: 2.97273.0052 - val_loss: 26538.8926 - val_RMSE_step_t_humidity: 282.5777 - val_RMSE_step_t_1_humidity: 282.4516 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.9414 - RMSE_step_t_humidity: 281.8454 - RMSE_step_t_1_humidity: 281.7919 - RMSE_step_t_wind_dir: 3.8305 - RMSE_step_t_1_wind_dir: 3.9819 - RMSE_step_t_wind_speed: 2.7877 - RMSE_step_t_1_wind_speed: 2.96472.9727 - val_loss: 26538.8887 - val_RMSE_step_t_humidity: 282.5777 - val_RMSE_step_t_1_humidity: 282.4515 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 89/100
10/10 [==============================] - ETA: 0s - loss: 26490.9414 - RMSE_step_t_humidity: 281.8454 - RMSE_step_t_1_humidity: 281.7919 - RMSE_step_t_wind_dir: 3.8305 - RMSE_step_t_1_wind_dir: 3.9819 - RMSE_step_t_wind_speed: 2.7877 - RMSE_step_t_1_wind_speed: 2.96472.9727 - val_loss: 26538.8887 - val_RMSE_step_t_humidity: 282.5777 - val_RMSE_step_t_1_humidity: 282.4515 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.6719 - RMSE_step_t_humidity: 281.7892 - RMSE_step_t_1_humidity: 281.7636 - RMSE_step_t_wind_dir: 3.8418 - RMSE_step_t_1_wind_dir: 4.0048 - RMSE_step_t_wind_speed: 2.7863 - RMSE_step_t_1_wind_speed: 2.99382.9647 - val_loss: 26538.8828 - val_RMSE_step_t_humidity: 282.5777 - val_RMSE_step_t_1_humidity: 282.4515 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 90/100
10/10 [==============================] - ETA: 0s - loss: 26491.6719 - RMSE_step_t_humidity: 281.7892 - RMSE_step_t_1_humidity: 281.7636 - RMSE_step_t_wind_dir: 3.8418 - RMSE_step_t_1_wind_dir: 4.0048 - RMSE_step_t_wind_speed: 2.7863 - RMSE_step_t_1_wind_speed: 2.99382.9647 - val_loss: 26538.8828 - val_RMSE_step_t_humidity: 282.5777 - val_RMSE_step_t_1_humidity: 282.4515 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.0645 - RMSE_step_t_humidity: 281.8696 - RMSE_step_t_1_humidity: 281.8729 - RMSE_step_t_wind_dir: 3.8451 - RMSE_step_t_1_wind_dir: 4.0186 - RMSE_step_t_wind_speed: 2.7634 - RMSE_step_t_1_wind_speed: 2.99142.9938 - val_loss: 26538.8691 - val_RMSE_step_t_humidity: 282.5776 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 91/100
10/10 [==============================] - ETA: 0s - loss: 26491.0645 - RMSE_step_t_humidity: 281.8696 - RMSE_step_t_1_humidity: 281.8729 - RMSE_step_t_wind_dir: 3.8451 - RMSE_step_t_1_wind_dir: 4.0186 - RMSE_step_t_wind_speed: 2.7634 - RMSE_step_t_1_wind_speed: 2.99142.9938 - val_loss: 26538.8691 - val_RMSE_step_t_humidity: 282.5776 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.0137 - RMSE_step_t_humidity: 281.6836 - RMSE_step_t_1_humidity: 281.7643 - RMSE_step_t_wind_dir: 3.8811 - RMSE_step_t_1_wind_dir: 4.0874 - RMSE_step_t_wind_speed: 2.7685 - RMSE_step_t_1_wind_speed: 2.98172.9914 - val_loss: 26538.8672 - val_RMSE_step_t_humidity: 282.5776 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 92/100
10/10 [==============================] - ETA: 0s - loss: 26491.0137 - RMSE_step_t_humidity: 281.6836 - RMSE_step_t_1_humidity: 281.7643 - RMSE_step_t_wind_dir: 3.8811 - RMSE_step_t_1_wind_dir: 4.0874 - RMSE_step_t_wind_speed: 2.7685 - RMSE_step_t_1_wind_speed: 2.98172.9914 - val_loss: 26538.8672 - val_RMSE_step_t_humidity: 282.5776 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.9277 - RMSE_step_t_humidity: 281.8347 - RMSE_step_t_1_humidity: 281.7995 - RMSE_step_t_wind_dir: 3.8853 - RMSE_step_t_1_wind_dir: 4.0468 - RMSE_step_t_wind_speed: 2.7938 - RMSE_step_t_1_wind_speed: 2.99482.9817 - val_loss: 26538.8613 - val_RMSE_step_t_humidity: 282.5775 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 93/100
10/10 [==============================] - ETA: 0s - loss: 26490.2930 - RMSE_step_t_humidity: 281.8096 - RMSE_step_t_1_humidity: 281.7537 - RMSE_step_t_wind_dir: 3.8374 - RMSE_step_t_1_wind_dir: 3.9765 - RMSE_step_t_wind_speed: 2.7742 - RMSE_step_t_1_wind_speed: 2.97102.9948 - val_loss: 26538.8535 - val_RMSE_step_t_humidity: 282.5775 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 94/100
10/10 [==============================] - ETA: 0s - loss: 26490.2930 - RMSE_step_t_humidity: 281.8096 - RMSE_step_t_1_humidity: 281.7537 - RMSE_step_t_wind_dir: 3.8374 - RMSE_step_t_1_wind_dir: 3.9765 - RMSE_step_t_wind_speed: 2.7742 - RMSE_step_t_1_wind_speed: 2.97102.9948 - val_loss: 26538.8535 - val_RMSE_step_t_humidity: 282.5775 - val_RMSE_step_t_1_humidity: 282.4514 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26489.8301 - RMSE_step_t_humidity: 281.9882 - RMSE_step_t_1_humidity: 281.9755 - RMSE_step_t_wind_dir: 3.9026 - RMSE_step_t_1_wind_dir: 4.0560 - RMSE_step_t_wind_speed: 2.8049 - RMSE_step_t_1_wind_speed: 3.01812.9710 - val_loss: 26538.8516 - val_RMSE_step_t_humidity: 282.5775 - val_RMSE_step_t_1_humidity: 282.4513 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 95/100
 9/10 [==========================>...] - ETA: 0s - loss: 26483.3477 - RMSE_step_t_humidity: 281.8250 - RMSE_step_t_1_humidity: 281.7445 - RMSE_step_t_wind_dir: 3.8737 - RMSE_step_t_1_wind_dir: 4.0457 - RMSE_step_t_wind_speed: 2.7867 - RMSE_step_t_1_wind_speed: 2.99753.0181 - val_loss: 26538.8457 - val_RMSE_step_t_humidity: 282.5775 - val_RMSE_step_t_1_humidity: 282.4513 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 96/100
 9/10 [==========================>...] - ETA: 0s - loss: 26483.3477 - RMSE_step_t_humidity: 281.8250 - RMSE_step_t_1_humidity: 281.7445 - RMSE_step_t_wind_dir: 3.8737 - RMSE_step_t_1_wind_dir: 4.0457 - RMSE_step_t_wind_speed: 2.7867 - RMSE_step_t_1_wind_speed: 2.99753.0181 - val_loss: 26538.8457 - val_RMSE_step_t_humidity: 282.5775 - val_RMSE_step_t_1_humidity: 282.4513 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
[34m[1mwandb[39m[22m: Adding directory to artifact (/home/aireandino/AI/wandb/run-20240307_180132-0zly269s/files/model-best)... Done. 0.0s
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26491.6055 - RMSE_step_t_humidity: 281.6449 - RMSE_step_t_1_humidity: 281.6578 - RMSE_step_t_wind_dir: 3.8473 - RMSE_step_t_1_wind_dir: 4.0366 - RMSE_step_t_wind_speed: 2.7703 - RMSE_step_t_1_wind_speed: 2.98212.9841 - val_loss: 26538.8379 - val_RMSE_step_t_humidity: 282.5774 - val_RMSE_step_t_1_humidity: 282.4513 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 97/100
10/10 [==============================] - ETA: 0s - loss: 26491.6055 - RMSE_step_t_humidity: 281.6449 - RMSE_step_t_1_humidity: 281.6578 - RMSE_step_t_wind_dir: 3.8473 - RMSE_step_t_1_wind_dir: 4.0366 - RMSE_step_t_wind_speed: 2.7703 - RMSE_step_t_1_wind_speed: 2.98212.9841 - val_loss: 26538.8379 - val_RMSE_step_t_humidity: 282.5774 - val_RMSE_step_t_1_humidity: 282.4513 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6037 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.4062 - RMSE_step_t_humidity: 281.7980 - RMSE_step_t_1_humidity: 281.8160 - RMSE_step_t_wind_dir: 3.8742 - RMSE_step_t_1_wind_dir: 4.0509 - RMSE_step_t_wind_speed: 2.7866 - RMSE_step_t_1_wind_speed: 3.01112.9821 - val_loss: 26538.8340 - val_RMSE_step_t_humidity: 282.5774 - val_RMSE_step_t_1_humidity: 282.4512 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 98/100
10/10 [==============================] - ETA: 0s - loss: 26490.4062 - RMSE_step_t_humidity: 281.7980 - RMSE_step_t_1_humidity: 281.8160 - RMSE_step_t_wind_dir: 3.8742 - RMSE_step_t_1_wind_dir: 4.0509 - RMSE_step_t_wind_speed: 2.7866 - RMSE_step_t_1_wind_speed: 3.01112.9821 - val_loss: 26538.8340 - val_RMSE_step_t_humidity: 282.5774 - val_RMSE_step_t_1_humidity: 282.4512 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26489.3926 - RMSE_step_t_humidity: 281.9700 - RMSE_step_t_1_humidity: 281.9201 - RMSE_step_t_wind_dir: 3.9168 - RMSE_step_t_1_wind_dir: 4.0903 - RMSE_step_t_wind_speed: 2.8134 - RMSE_step_t_1_wind_speed: 3.04063.0111 - val_loss: 26538.8203 - val_RMSE_step_t_humidity: 282.5774 - val_RMSE_step_t_1_humidity: 282.4512 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 99/100
10/10 [==============================] - ETA: 0s - loss: 26489.3926 - RMSE_step_t_humidity: 281.9700 - RMSE_step_t_1_humidity: 281.9201 - RMSE_step_t_wind_dir: 3.9168 - RMSE_step_t_1_wind_dir: 4.0903 - RMSE_step_t_wind_speed: 2.8134 - RMSE_step_t_1_wind_speed: 3.04063.0111 - val_loss: 26538.8203 - val_RMSE_step_t_humidity: 282.5774 - val_RMSE_step_t_1_humidity: 282.4512 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776
/usr/lib/python3/dist-packages/keras/engine/training.py:3000: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
10/10 [==============================] - ETA: 0s - loss: 26490.6426 - RMSE_step_t_humidity: 281.7834 - RMSE_step_t_1_humidity: 281.7652 - RMSE_step_t_wind_dir: 3.7856 - RMSE_step_t_1_wind_dir: 3.9626 - RMSE_step_t_wind_speed: 2.7469 - RMSE_step_t_1_wind_speed: 2.95643.0406 - val_loss: 26538.8145 - val_RMSE_step_t_humidity: 282.5773 - val_RMSE_step_t_1_humidity: 282.4511 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776
Epoch 100/100
10/10 [==============================] - 4s 435ms/step - loss: 26490.6426 - RMSE_step_t_humidity: 281.7834 - RMSE_step_t_1_humidity: 281.7652 - RMSE_step_t_wind_dir: 3.7856 - RMSE_step_t_1_wind_dir: 3.9626 - RMSE_step_t_wind_speed: 2.7469 - RMSE_step_t_1_wind_speed: 2.9564 - val_loss: 26538.8105 - val_RMSE_step_t_humidity: 282.5773 - val_RMSE_step_t_1_humidity: 282.4511 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776
10/10 [==============================] - 4s 435ms/step - loss: 26490.6426 - RMSE_step_t_humidity: 281.7834 - RMSE_step_t_1_humidity: 281.7652 - RMSE_step_t_wind_dir: 3.7856 - RMSE_step_t_1_wind_dir: 3.9626 - RMSE_step_t_wind_speed: 2.7469 - RMSE_step_t_1_wind_speed: 2.9564 - val_loss: 26538.8105 - val_RMSE_step_t_humidity: 282.5773 - val_RMSE_step_t_1_humidity: 282.4511 - val_RMSE_step_t_wind_dir: 2.9124 - val_RMSE_step_t_1_wind_dir: 2.9014 - val_RMSE_step_t_wind_speed: 2.6036 - val_RMSE_step_t_1_wind_speed: 2.5776